Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -167,6 +167,7 @@ The secret contains credentials used by the application pod to access the bucket
 The ConfigMap contains bucket endpoint information and is also consumed by the pod.
 See the [Object Bucket Claim Documentation](ceph-object-bucket-claim.md) for more details on the `CephObjectBucketClaims`.
 
+
```

Comment:
Convention has been to use only a single line between the end of a section and the start of the next header in Rook docs.